onnx/test/shape_inference_test.py

```diff
@@ -37,16 +37,20 @@ def _make_graph(self,
                 input_value_infos.append(seed_value_info)
             else:
                 value_info.append(seed_value_info)
-                input_value_infos.append(make_tensor_value_info('SEED_' + seed_name, proto_type, ()))
-                input_value_infos.append(make_tensor_value_info('UNKNOWN_SHAPE_' + seed_name, TensorProto.INT64, ()))
-                nodes[:0] = [make_node("Reshape", ['SEED_' + seed_name, 'UNKNOWN_SHAPE_' + seed_name], [seed_name])]
+                input_value_infos.append(make_tensor_value_info(
```

Comment:
Unfortunately, the formatting changes makes it hard to find which lines of code changed. Is it possible to undo the formatting changes?

Reply:
I must have had "format on save" option on. I reverted the changes, will modify/add shape inference tests soon.